app/assets/javascripts/app/phone-internationalization.js

```diff
@@ -74,14 +76,56 @@ const updateOTPDeliveryMethods = () => {
   }
 };
 
+const internationalCodeFromPhone = (phone) => {
+  const match = phone.match(INTERNATIONAL_CODE_REGEX);
+  if (match) {
+    return match[1] || match[2];
+  }
+  return '1';
+};
+
+const updateInternationalCodeSelection = () => {
+  const phoneInput = document.querySelector('#two_factor_setup_form_phone');
+  const phone = phoneInput.value;
+  const internationalCode = internationalCodeFromPhone(phone);
+  const option = document.querySelector(`[data-country-code='${internationalCode}'`);
```

Comment:
This line is missing a closing square bracket after the closing single quote. It's what's causing the tests to fail. Interestingly, Chrome doesn't complain about this, but Safari does.